charts/argo-events/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A Helm chart to install Argo-Events in k8s Cluster
 name: argo-events
-version: 1.2.3
+version: 1.2.4
```

Comment:
This is technically a breaking change the way it's currently done if anyone is already using `serviceAccount`.